_includes/reference/cli-template.md

```diff
@@ -1,17 +1,15 @@
 {% assign app = file.application %}
 
 {% assign commands = file.commands %}
-{% if app %}
-# {{ app.name }} (version {{ app.version }})
-{% else %}
-# Magento CLI
-{% endif %}
 
-This reference contains {{ commands | size }} commands available through the `bin/magento` command-line tool.
-The initial list is auto generated using the `bin/magento list` command at the {{ site.data.var.ce }} edition. Use the ["Add CLI commands"]({{ page.baseurl }}/extension-dev-guide/cli-cmds/cli-add.html) guide to add a custom Magento CLI command.
+This reference contains {{ commands | size }} commands available through the `{{ tool }}` command-line tool.
+The initial list is auto generated using the `{{ tool }} list` command at the {{ edition }} edition.
+
+{{ intro }}
 
-{: .bs-callout-info }
-You can call Magento CLI commands using shortcuts instead of the full command name. For example, you can call `bin/magento setup:upgrade` using `bin/magento s:up`, `bin/magento s:upg`, etc. See [shortcut syntax](https://symfony.com/doc/current/components/console/usage.html#shortcut-syntax) to understand how to use shortcuts with any Magento CLI command.
+{% if app.version %}
+version {{ app.version }}
```

Comment:
I think this line is important, but I would like to see it more pronounced or more obvious. Maybe we could create a summary list of sorts. For example:

**Tool**: 
**Version**:

Reply:
Good point, I didn't finish with 'version'.